runtimes/protocol/identity-management.ts

```diff
@@ -192,46 +198,19 @@ export const invalidateSsoTokenRequestType = new ProtocolRequestType<
     void
 >('aws/identity/invalidateSsoToken')
 
-// updateSsoTokenManagement
-export interface UpdateSsoTokenManagementParams {
-    readonly ssoTokenId: SsoTokenId
-    autoRefresh?: boolean // no change if not set
-    changeNotifications?: boolean // no change if not set
-}
-
-export interface UpdateSsoTokenManagementResult {
-    readonly ssoTokenId: SsoTokenId // same as supplied request params value
-    readonly autoRefresh: boolean // returns state after call
-    readonly changeNotifications: boolean // returns state after call
-}
-
-// Potential error codes: E_UNKNOWN | E_TIMEOUT | E_INVALID_TOKEN
-export const updateSsoTokenManagementRequestType = new ProtocolRequestType<
-    UpdateSsoTokenManagementParams,
-    UpdateSsoTokenManagementResult,
-    never,
-    AwsResponseError,
-    void
->('aws/identity/updateSsoTokenManagement')
-
 // ssoTokenChanged
-export type Created = 'Created'
 export type Refreshed = 'Refreshed'
-export type Expired = 'Expired'
```

Comment:
From the description `Expired` wasn't meant to be removed? Or missed removing in `SsoTokenChangedKind `